addons/mail/static/src/models/thread_cache/thread_cache.js

```diff
@@ -512,6 +524,8 @@ function factory(dependencies) {
             compute: '_computeMessages',
             dependencies: [
                 'fetchedMessages',
+                'messageSender',
+                'senderPendingMessagesToBeSent',
```

Comment:
This should also depend on `senderErrorNotDelivred` now (or whatever it is after rename suggestion)